### DISCUSSION TITLE
any keyword

### DISCUSSION THREAD

**Discussion by rraammiinn** [vlang/v#17554] [Ideas]
can you add any keyword as a syntax sugar for empty interface to get rid of declaring it like this ( #14162 ) ?

**Comment by JalonSolov** [comment]
It's already in the ROADMAP for v0.4.

https://github.com/vlang/v/blob/master/ROADMAP.md

**Reply by rraammiinn** [reply]
thanks , that's all I wanted .
plus shadowing that you disagreed of course .

**Comment by rraammiinn** [comment]
I know you will not agree but it's cool to make type annotation in functions optional and unannotated parameters resolve to any .

**Reply by bogen85** [reply]
> I know you will not agree but it's cool to make type annotation in functions optional and unannotated parameters resolve to any .

That is in direct opposition to being a strongly and statically typed language, which V is.

**Reply by rraammiinn** [reply]
but it can be enabled using flags or have strict model like javascript .
v supports multiple types of memory management , why not support multiple styles of programming .
and there are ides and linters too .

**Reply by JalonSolov** [reply]
There are multiple programming languages for a reason.  Use the one that works the best for your project, rather than trying to force one to do everything.
